tests/lib/prepare-restore.sh

```diff
@@ -221,7 +221,7 @@ prepare_project() {
     # we install chrony on xenial (as its also used in later distros), so the
     # ntp service was in conflict, degrading the systemd unit and sometimes breaking
     # NTP syncs
-    if os.query is-xenial; then
+    if os.query is-xenial && systemctl is-enabled ntp; then
```

Comment:
Can you drop the line that pre-installs ntp in .image-garden.mk. This is the only reason I chose to include it -- to exclude all other changes to spread.yaml while introducing the garden backend.

Reply:
updated